src/Microsoft/MicrosoftAvatar.php

```diff
@@ -32,7 +32,7 @@ public function setResponse(Response $response)
      *
      * @return string
      */
-    public function getContentType(): string
+    public function getContentType()
```

Comment:
i'm pretty sure 7.0 supports this?

Reply:
It does not match composer dependencies constraints: https://github.com/SocialiteProviders/Providers/blob/master/src/Microsoft/composer.json#L10

This package is supposed to work with `PHP 5.6`

And it is not the case now